lib/galaxy/config.py

```diff
@@ -43,6 +43,7 @@
     job_config_file=['config/job_conf.xml', 'job_conf.xml'],
     tool_destinations_config_file=['config/tool_destinations.yml', 'config/tool_destinations.yml.sample'],
     job_metrics_config_file=['config/job_metrics_conf.xml', 'job_metrics_conf.xml', 'config/job_metrics_conf.xml.sample'],
+    error_report_file=['config/error_report.xml', 'error_report.xml', 'config/error_report.xml.sample'],
```

Comment:
I'd remove this middle entry - we allow them to sit in ``$GALAXY_ROOT`` for some older configs for legacy reasons - they were there before we introduced the ``config/`` directory - we don't want new config files in ``$GALAXY_ROOT``.